tests/conftest.py

```diff
@@ -24,6 +28,23 @@ def dbt_profile_target():
     
     if os.getenv('DBT_TERADATA_PASSWORD'):
         password=os.getenv('DBT_TERADATA_PASSWORD')
+
+    
+    with teradatasql.connect (host=hostname,user=username,password=password) as con:
+        with con.cursor () as cur:
+            TEST_USER_ENV_VARS = ["DBT_TEST_USER_1", "DBT_TEST_USER_2", "DBT_TEST_USER_3"]
+            for env_var in TEST_USER_ENV_VARS:
+                user_name = os.getenv(env_var)
+                if user_name:
+                    try:
+                        print("Creating user by submitting below SQL:")
```

Comment:
instead of print, logger.debug can't be used?

Reply:
No Satish. No logger implementation in conftest.py

Reply:
we can add this statement. dbt/adapters/teradata/connections.py has logger statemeents. This may help mohan.
logger = AdapterLogger("teradata")

Reply:
conftest.py runs on VM provision in the GitHub CI integration which runs outside of dbt with pytest module

Reply:
Removing print()